rewriter/Prop.cc

```diff
@@ -586,6 +586,7 @@ void Prop::run(core::MutableContext ctx, ast::ClassDef *klass) {
     UnorderedMap<void *, vector<ast::ExpressionPtr>> replaceNodes;
     replaceNodes.reserve(klass->rhs.size());
     vector<PropInfo> props;
+    UnorderedMap<core::NameRef, uint32_t> seenProps;
```

Comment:
It seems like this `uint32_t` is unused? Is there a reason this isn't an `UnorderedSet`?

Reply:
It is used in the error case below to point at the previous definition.

Reply:
It's used when we construct the error, to index into `props` and find the first occurrence.

Reply:
Oh thanks, I thought I searched for `it` but I must have typo'd it